locale/en/download/package-manager.md

```diff
@@ -20,6 +20,7 @@ title: Installing Node.js via package manager
 * [NetBSD](#netbsd)
 * [Nodenv](#nodenv)
 * [nvm](#nvm)
+* [fnm](#fnm)
```

Comment:
The list is in case-insensitive alphabetical order, so this should be added after line 15 instead of here.

Reply:
Thanks, sorry I missed the order logic, changing 🙏

Reply:
Done ✅